sqlest/src/main/scala/sqlest/sql/base/BaseStatementBuilder.scala

```diff
@@ -18,8 +18,13 @@ package sqlest.sql.base
 
 import sqlest.ast._
 import sqlest.ast.operations.ColumnOperations._
+import org.joda.time.{ LocalDate, DateTime }
+import org.joda.time.format.{ DateTimeFormat, DateTimeFormatter }
 
 trait BaseStatementBuilder {
+  val dateTimeFormat: DateTimeFormatter = DateTimeFormat.forPattern("yyyy-MM-dd-HH.mm.ss.SSSSSS")
+  val localDateFormat: DateTimeFormatter = DateTimeFormat.forPattern("yyyy-MM-dd")
```

Comment:
I probably should have used [ISODateTimeFormat](http://www.joda.org/joda-time/apidocs/org/joda/time/format/ISODateTimeFormat.html)